tests/monster_test_generated.ts

```diff
@@ -3633,7 +3635,9 @@ static addV8(builder:flatbuffers.Builder, v8Offset:flatbuffers.Offset) {
  * @param Array.<number> data
  * @returns flatbuffers.Offset
  */
-static createV8Vector(builder:flatbuffers.Builder, data:number[] | Uint8Array):flatbuffers.Offset {
+static createV8Vector(builder:flatbuffers.Builder, data:number[]|Int8Array):flatbuffers.Offset;
+static createV8Vector(builder:flatbuffers.Builder, data:number[]|Uint8Array):flatbuffers.Offset;
```

Comment:
> Looks good! I would suggest adding a deprecation notice to uint version - we shouldn't really support it, as it does not make much sense.

Thanks, @krojew.
Concerning "deprecation notice", did you mean adding some comments in the generated code?
For example,
```diff
  static createV8Vector(builder:...|Int8Array):flatbuffers.Offset;
- static createV8Vector(builder:...|Uint8Array):flatbuffers.Offset;
+ /**
+  * @deprecated This Uint8Array overload will be removed in the future.
+  */
+ static createV8Vector(builder:...|Uint8Array):flatbuffers.Offset;
```

Reply:
Yes

Reply:
Okay. I've added the comment in 3a3aba0bba94f367f86d276448393c0979df3181.